src/index.js

```diff
@@ -1,63 +1,123 @@
+} from './crud.js';
+
+let tasks = [];
 
 const bookList = document.getElementById('bookList');
+const input = document.getElementById('task-input');
 
 const getLocalStorageTasks = () => {
-  const getData = localStorage.getItem('TasksList');
-  const data = JSON.parse(getData);
-  if (data) {
-    tasks = data;
+  const getData = localStorage.getItem('tasksList');
+  if (getData !== undefined) {
+    const data = JSON.parse(getData);
+    if (data) {
+      tasks = data;
+    } else {
+      tasks = [];
+    }
   }
 };
 
+const pushLocalStorageTasks = (tasks) => {
+  localStorage.setItem('tasksList', JSON.stringify(tasks));
+};
+
 const sortTasks = (tasksArray) => {
   tasksArray.sort((taskA, taskB) => taskA.index - taskB.index);
 };
 
 const populatelist = () => {
-  sortTasks(tasks);
+  sortTasks();
   bookList.innerHTML = '';
-  tasks.forEach((tasks) => {
-    if (tasks.completed === true) {
-      bookList.innerHTML += `
-    <li class="list-item">
-    <input type="checkbox" class="position-grid-start" id="${tasks.index}" checked/>
-      <p> ${tasks.description} </p>
+  if (!tasks) {
+    bookList.innerHTML += `
+    <li class="list-item" draggable="true">
+      <p > No Tasks </p>
+    <li>   
+    `;
+  } else {
+    tasks.forEach((tasks) => {
+      if (tasks.completed === true) {
+        bookList.innerHTML += `
+    <li class="list-item" draggable="true">
+      <input type="checkbox" class="position-grid-start" id="${tasks.index}" checked/>
+      <p contenteditable="true" class="editable"> ${tasks.description} </p>
       <i class="fas fa-ellipsis-v position-grid-end"></i>
+      <i id="${tasks.index}" class="fas fa-trash-alt position-grid-end delete-button"></i>
     <li>   
     `;
-    } else {
-      bookList.innerHTML += `
-    <li class="list-item">
+      } else {
+        bookList.innerHTML += `
+    <li class="list-item" draggable="true">
     <input type="checkbox" class="position-grid-start" id="${tasks.index}"/>
-      <p> ${tasks.description} </p>
+      <p contenteditable="true" class="editable"> ${tasks.description} </p>
       <i class="fas fa-ellipsis-v position-grid-end"></i>
+      <i id="${tasks.index}" class="fas fa-trash-alt position-grid-end delete-button"></i>
     <li>   
     `;
+      }
+    });
+  }
+  bookList.innerHTML += '<button id="clear-button" class="clear-button"> Clear All Completed </button>';
+  document.querySelector('.delete-button').addEventListener('click', (event) => {
+    const { target } = event;
+    if (target.classList.value.indexOf('delete') !== -1) {
+      const tasksArray = JSON.parse(localStorage.getItem('tasksList'));
+      // eslint-disable-next-line max-len
+      const index = tasksArray.findIndex((element) => element.index === parseInt(target.id, 10));
+      // deleteTask(index, tasksArray);
+      // representNewIndexes(tasksArray);
+      tasksArray.splice(index, 1);
+      localStorage.setItem('tasksList', JSON.stringify(tasksArray));
+      // location.reload();
```

Comment:
- [ ]  Let us fix eslint errors without disabling them, this makes your code looks unprofessional.
- [ ]  Remove the commented lines in the code that has been pushed.